core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/Primitive.kt

```diff
@@ -10,50 +10,23 @@ import ch.ergon.dope.validtype.NumberType
 import ch.ergon.dope.validtype.StringType
 import ch.ergon.dope.validtype.ValidType
 
-class Primitive<T : ValidType> : TypeExpression<T> {
-    private val dopeQuery: DopeQuery
+open class Primitive<T : ValidType>() : TypeExpression<T> {
+    private var queryString: String = ""
```

Comment:
This doesn't make sense. Now you can create a Primitive with an emtpy queryString. I would go for the following approach:
```
sealed class Primitive<T : ValidType>(val dopeQuery: DopeQuery) : TypeExpression<T> {
    override fun toDopeQuery() = dopeQuery
}
```
Primitve is sealed -> We can cerate out subclasses we want
Constructor want a dopeQuery -> We directly define in out subclasses the queryString and the params

Reply:
done